src/blog/_posts/2017-06-22-opera-46.md

```diff
@@ -0,0 +1,84 @@
+* The [Image Capture API](https://developers.google.com/web/updates/2016/12/imagecapture) now allows sites to take higher resolution images than before, providing full control over camera settings such as zoom, ISO, and white balance.
+* To provide enhanced privacy, CSS stylesheets can now specify their own [referrer policy](https://w3c.github.io/webappsec-referrer-policy/) via the HTTP header, rather than always inheriting the referrer policy of the document that originally referenced it.
+* [Touch events](https://developer.mozilla.org/en-US/docs/Web/API/Touch_events) are now aligned to [`requestAnimationFrame`](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame), ensuring that input is processed as part of the document lifecycle and creating a more efficient and adaptive input response.
+* The new [`worker-src`](https://w3c.github.io/webappsec-csp/#directive-worker-src) Content Security Policy directive restricts which URLs may be loaded as a [`Worker`](https://developer.mozilla.org/en-US/docs/Web/API/Worker), [`SharedWorker`](https://developer.mozilla.org/en-US/docs/Web/API/SharedWorker), or [`ServiceWorker`](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorker).
+
+## Deprecations and interoperability improvements
+
+* The `<dialog>` element has changed from display: inline to block by default to better align with the [spec](https://html.spec.whatwg.org/multipage/rendering.html#flow-content-3).
+* Following [removal](https://github.com/w3c/csswg-drafts/commit/2078b46218f7462735bb0b5107c9a3e84fb4c4b1) from the [Media Queries](https://www.w3.org/TR/css3-mediaqueries/) spec, support for `hover: on-demand` and `any-hover: on-demand` media queries have been removed.
+* To better align with spec and help avoid race conditions, [`decodeAudioData`](https://webaudio.github.io/web-audio-api/#widl-BaseAudioContext-decodeAudioData-Promise-AudioBuffer--ArrayBuffer-audioData-DecodeSuccessCallback-successCallback-DecodeErrorCallback-errorCallback) now detaches the given ArrayBuffer before decoding, removing all content from the object and making it unable to be reused or examined.
+* The `-internal-media-controls-cast-button` CSS selector has been removed in favor of the [Remote Playback API](https://w3c.github.io/remote-playback/).
+* The `-internal-media-controls-text-track-list*` CSS selectors have been removed in favor of custom-built video controls.
+* The `SVGTests.requiredFeatures` attribute has been deprecated following its removal from the [spec](https://github.com/w3c/svgwg/commit/9a30d01f6410dc516c5f874d71e957230a3448cd).
```

Comment:
s/deprecated/removed/ (deprecation was in Opera 41/Chromium 54, https://www.chromestatus.com/features/5720709590417408)